build-system/compile/bundles.config.js

```diff
@@ -256,24 +256,5 @@ exports.verifyExtensionBundles = function () {
       bundle.name,
       bundleString
     );
-    verifyBundle_(
-      'latestVersion' in bundle,
-      'latestVersion',
-      'is missing from',
-      bundle.name,
-      bundleString
-    );
-    const duplicates = exports.extensionBundles.filter(
-      (duplicate) => duplicate.name === bundle.name
-    );
-    verifyBundle_(
-      duplicates.every(
-        (duplicate) => duplicate.latestVersion === bundle.latestVersion
-      ),
-      'latestVersion',
-      'is not the same for all versions of',
-      bundle.name,
-      JSON.stringify(duplicates, null, 2)
```

Comment:
/cc @rileyajones who is reusing this logic for the bento build workflow. Now that we're reducing the number of checks here, perhaps there's scope to further simplify the entire verification logic for `bundles.config.js`?